config/marconi.mk

```diff
@@ -5,7 +5,7 @@ INCLUDE += -I$(NETCDF_INC) -I$(HDF5_INC)
 GLFLAGS  = -lm
 CC=icc
 MPICC=mpiicc
-OPT=-O3 -xHost
+OPT=-O3 -xHost -fma -xMIC-AVX512 -finline-functions -align
```

Comment:
From Intel's web page: _AVX-512F, AVX-512CD, AVX-512ER, and AVX-512PF are implemented in the Intel Xeon Phi processor x200 (code named Knights Landing)_ 
I agree to consider KNL an independent device since performance and configuration may differ from regular multicore processors.